pkgs/applications/audio/setbfree/default.nix

```
@@ -36,10 +36,9 @@ stdenv.mkDerivation  rec {
 
   doInstallCheck = true;
 
-  installCheckPhase = ''(
-    set -x
-    test -e $out/bin/setBfreeUI
-  )'';
+  installCheckPhase = ''
+    (set -x; test -e $out/bin/setBfreeUI)
```

Why not like in the other cases:

```suggestion
    (
      set -x
      test -e $out/bin/setBfreeUI
    )
```